### PR TITLE
Signup flow styles

### PR DESCRIPTION
**Pull request by sandersonet**
Improved styles for
* Login
* Signup
* Create app
* Add payment info

Latest on aptible-sass#diesel required.

### CODE REVIEW

**Comment by sandersonet** [comment]
For you @mixonic

File: app/welcome/first-app/route.js [hunk 1]
```diff
@@ -1,23 +1,8 @@
...
-      return firstApp;
-    } else {
-      return {
-        appHandle: '',
-        dbHandle: '',
-        diskSize: 10,
-        dbType: null
-      };
-    }
-  },
```
Comment (mixonic) [inline app/welcome/first-app/route.js]: This code was responsible for restoring the app and db as entered if a user returned to the wizard. And in general it provides defaults for the initial screens.

It seems like it should not be dropped?

Comment (sandersonet) [inline app/welcome/first-app/route.js]: I moved it to the parent route.

File: app/welcome/payment-info/route.js [hunk 2]
```diff
@@ -2,9 +2,6 @@ import Ember from 'ember';
 import { createStripeToken } from '../../utils/stripe';
 
 export default Ember.Route.extend({
-  model: function() {
-    return {};
-  },
```
Comment (mixonic) [inline app/welcome/payment-info/route.js]: This was also intentional- to return a blank object.

File: app/welcome/payment-info/controller.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Ember from "ember";
+
+export default Ember.Controller.extend({
+  months: ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'],
+  years: ['2015', '2016', '2017', '2018', '2019', '2020'],
+  plans: ['development', 'pilot', 'platform', 'production']
```
Comment (mixonic) [inline app/welcome/payment-info/controller.js]: When details like this are stored on a controller, I usually end up creating components. For example, `months` is used in a select box. So I suggest a `select-months` component that internalizes the list of options. This is much more re-usable and semantic.

File: app/welcome/first-app/route.js (hunk 1 above)
Comment (mixonic) [inline app/welcome/first-app/route.js]: That means the app/db info object is now shared with the payment info object? What is the motivation? IMO it is more flexible/isolated to keep them separate.

**Comment by sandersonet** [comment]
@mixonic Updated per your feedback.  I'm still not sure about the `model` function in `welcome/payment-info/route.js`.  The goal here is to be able to use the app/db name created in the prior step in the copy on the right.  Thoughts?  Thanks for your help.

**Pull request merged**